clash-prelude/src/Clash/Class/Resize.hs

```diff
@@ -117,3 +120,57 @@ checkedTruncateB ::
   f (a + b) -> f a
 checkedTruncateB v =
   checkIntegral (Proxy @(f a)) v `seq` truncateB v
+
+-- | Helper function of 'maybeFromIntegral', 'maybeResize' and 'maybeTruncateB'
+maybeIntegral ::
+  forall a b.
+  (Integral a, Integral b, Bounded b) =>
+  Proxy b ->
+  a -> Bool
+maybeIntegral Proxy v =
+     toInteger v < toInteger (maxBound @b)
+  && toInteger v > toInteger (minBound @b)
+
+-- | Like 'fromIntegral', but returns 'Nothing' if /a/ is out of bounds for /b/.
+-- Useful when you do not know /a/ can be out of bounds, and would like to have
+-- your assumptions checked.
+
```

Comment:
The Haddock for `maybeFromIntegral` is not generated, and that's probably because of this blank line. This blank line means that what follows the blank line is a regular Haskell comment rather than a Haddock comment. I would have expected the Haddock to still be associated with the function, but apparently it's cut off and lost?

TL;DR: Remove the blank line :-). And always check the Haddock output :-).

To view the Haddock, you can click on the _Details_ button next to the _ci/gitlab/gitlab.com_ CI test result, and then click on _haddock_ (in the column _test_), and then click the big _Browse_ button in the rightmost column of that webpage.

Or just generate Haddock on your own system of course, before you run CI.